github_rules/github_repo_created.yml

```diff
@@ -16,7 +16,7 @@ Tests:
     Log:
       {
         "actor": "cat",
-        "action": "repo.created",        
+        "action": "repo.create",        
```

Comment:
Confirmed: https://docs.github.com/en/enterprise-server@3.4/admin/monitoring-activity-in-your-enterprise/reviewing-audit-logs-for-your-enterprise/audit-log-events-for-your-enterprise#repo-category-actions